src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+
+    <main>
+        <section id="App">
+            <div class="title">
+                <h1>Leaderboard</h1>
+            </div>
+            <div class="board">
+                <div class="scoresTable">
+                    <div class="tableHead">
+                        <div class="tableHead_title">
+                            <h3>Recent Scores</h3>
+                        </div>
+                        <div class="tableHead_button">
+                            <button>Refresh</button>
+                        </div>
+                    </div>
+                    <div class="tableBody">
+
+                    </div>
+
+                </div>
+                <div class="scoreSubmit">
+                    <div class="scoreSubmit_title">
+                        <h3>Add your score</h3>
+                    </div>
+                    <div class="scoreSubmit_input1">
+                        <input type="text" placeholder="Your name">
+                    </div>
+                    <div class="scoreSubmit_input2">
+                        <input type="number" placeholder="Your score">
+                    </div>
+                    <div class="scoreSubmit_button">
+                        <button>Submit</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>    
+  
+  <script src="index.js" type="module" ></script>     
```

Comment:
[Optional] This script tag will be inserted by webpack automatically when you build your script. So you can remove it to make your code cleaner.